core/web-assets/src/main/assets/js/apps/onms-daemons/index.js

```diff
@@ -0,0 +1,106 @@
+// noinspection JSAnnotator
+const angular = require('vendor/angular-js');
+const elementList = require('../onms-elementList/lib/elementList');
+require('../../lib/onms-pagination');
+require('../../lib/onms-http');
+require('angular-bootstrap-confirm');
+require('angular-bootstrap-toggle/dist/angular-bootstrap-toggle');
+require('angular-bootstrap-toggle/dist/angular-bootstrap-toggle.css');
+require('angular-ui-router');
+
+const indexTemplate = require('./index.html');
+
+(function () {
+    'use strict';
+
+    var MODULE_NAME = 'onms.daemons';
+
+    angular.module(MODULE_NAME, [
+        'angular-loading-bar',
+        'ngResource',
+        'ui.router',
+        'ui.bootstrap',
+        'ui.checkbox',
+        'ui.toggle',
+        'onms.http',
+        'onms.elementList',
+        'mwl.confirm',
+        'onms.pagination'
```

Comment:
Do we need all these modules?
It looks to me, they are not all used, can we only include those which are actually required?